packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -172,10 +178,12 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
       ..begin = Colors.transparent
       ..end = widget.backgroundColor ?? Colors.transparent;
 
+    final bool closed = !_isExpanded && _controller.value == 0.0;
```

Comment:
Yes, isDismissed is better. I'd been thrown by the AnimationController's toString() method which said that it was "paused" which I'd misinterpreted as some other state, not completed or dismissed.